cdap-data-fabric/src/test/java/co/cask/cdap/data2/util/hbase/HBaseVersionTest.java

```diff
@@ -74,6 +74,14 @@ public void testHBaseVersionToCompatMapping() throws ParseException {
     assertCompatModuleMapping(HBaseVersion.Version.HBASE_11, "1.2.0-IBM-7");
     assertCompatModuleMapping(HBaseVersion.Version.UNKNOWN, "1.3.0");
 
```

Comment:
Can add `assertCompatModuleMapping(HBaseVersion.Version. HBASE_94, "0.94.15-cdh4.7.1");` for cdh 4.7.1 and `assertCompatModuleMapping(HBaseVersion.Version. HBASE_98, "0.98.12-hadoop2");` for Bigtop 1.1.0